src/js/utils/behavior.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+const assign = require('object-assign');
+const forEach = require('array-foreach');
+const Behavior = require('receptor/behavior');
+
+const sequence = function () {
```

Comment:
I think you could use the [spread operator](https://babeljs.io/learn-es2015/#ecmascript-2015-features-default-rest-spread) to write this line as

```js
const sequence = function(...seq) {
```

and remove the following line (`const seq = [].slice.call(arguments);`